src/style.css

```diff
@@ -1,3 +1,66 @@
-body {
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap';
+
+:root {
+  --blue-color: #00acdc;
+  --white-color: #fff;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+}
```

Comment:
[checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md): Don't use * selector (it impacts performance)